cypress/support/pages/login_management/pageInvalidCredentials.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+class Invalid_Credentials {
+  static #emailInputBox = '[data-testid="username-email"]'
+  static #passwordField = '[data-testid = "password-email"]'
+  static #notificationBannerField = '[data-testid="notification"]'
+  static #loginFormSelector = 'form.login-form'
+  static #inputFieldInvalidPseudoSelector = 'input:invalid'
+
+  static #testData = {
+    usernameText:
+      'Fugiat consectetur deserunt officia velit. Dolore laboris incididunt consequat pariatur officia.',
+    emailText: 'robot.test+.exe@@@@@@@gmx.de',
+    invalidPassword:
+      'sc9lwOX#Z!ImcKVp66SP9ag$RvEX00nhR&Vn@dIW@hhREU||Zhbhbhu&&&$)Uhbwhbdbb|||',
+    errorMessageText: 'Login fehlgeschlagen.'
+  }
+
+  emailFieldIsVisibleAndEmpty () {
+    cy.get(Invalid_Credentials.#loginFormSelector).within(() => {
+      cy.get(Invalid_Credentials.#emailInputBox)
+        .should('be.visible')
+        .should('have.value', '')
+        .then(el => {
+          expect(el[0].checkValidity()).to.be.false
+        })
+      cy.get(Invalid_Credentials.#inputFieldInvalidPseudoSelector).should(
+        'have.length',
+        2
+      )
+    })
+  }
+
+  enterInvalidEmailOrUsername (usernameOrEmail) {
```

Comment:
the value of usernameOrEmail is true or false.
That can be used, but it is a bit complex to read because you have to investigate if true means username or Email. Perhaps a comment could help.